src/components/create-from-builder/create-from-builder-configure.html

```diff
@@ -90,10 +90,11 @@
 
     <!--
       Only show the link for existing projects. It will be broken for new
-      projects.  Make `invisible` instead of using an ng-if so that the dialog
-      doesn't resize when the message appears.
+      projects.  Use class `invisible` when the project list is still loading
+      so the dialog doesn't resize.
     -->
-    <div ng-class="{ invisible: !$ctrl.selectedProject.metadata.uid || !$ctrl.istag }" class="form-group">
+    <div ng-hide="($ctrl.selectedProject && !$ctrl.selectedProject.metadata.uid) || !$ctrl.istag"
```

Comment:
this also changes the behavior from invisible to hidden when `!$ctrl.istag` , just making sure that was intentional